clients/python/src/raystack/stencil.py

```diff
@@ -0,0 +1,30 @@
+from .store import Store
+from schedule import Scheduler
+from google.protobuf.message import Message
+
+class MultiUrlClient:
+    def __init__(self, urls:list, interval=3600, auto_refresh=False) -> None:
+        self._store = Store()
+        self._urls = urls
+        self._interval = interval
+        self._auto_refresh = auto_refresh
+        self._scheduler = Scheduler()
+        if self._auto_refresh:
+            self._scheduler.every(self._interval).seconds.do(self.refresh)
+        #TODO: check whether scheduler executed immediatelly or not
```

Comment:
When are we going to test this ?

Reply:
checked it, it starts after 1st schedule